### PR TITLE
fix: add underlayColor to SocialIconProps interface

### PR DESCRIPTION
**Pull request by Rawphs**
Added `underlayColor` to the `SocialIconProps` interface.

I wasn't sure as for what to write as a default, since it is only white if the button is light and the default color varies according to the button type.

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -1771,6 +1771,13 @@ export interface SocialIconProps {
    * @default false
    */
   loading?: boolean;
+  
+  /**
+   * Specify underlayColor for TouchableHighlight
+   *
+   * @default colors[type] || 'white'
```
Comment (iRoachie) [inline src/index.d.ts]:
```suggestion
   * @default 'white' if `light` prop is true, otherwise defaults to icon color.
```

**Comment by iRoachie** [comment]
Hey there @Rawphs! Thanks for sending this in and congrats on your first PR to React Native Elements.

There's just one change to the description I'd like to make, but other than that this is fine!

File: src/index.d.ts (hunk 1 above)
Comment (Rawphs) [reply src/index.d.ts]: @iRoachie Done!

**Comment by Rawphs** [comment]
cmfkfgp - imgur

**Comment by iRoachie** [comment]
Thanks! 💯

**Pull request merged**